python/mxnet/image/image.py

```diff
@@ -1091,7 +1093,7 @@ def __init__(self, batch_size, data_shape, label_width=1,
                 imgkeys = []
                 for line in iter(fin.readline, ''):
                     line = line.strip().split('\t')
-                    label = nd.array([float(i) for i in line[1:-1]])
+                    label = nd.array([i for i in line[1:-1]], dtype=dtype)
```

Comment:
[i for i in line[1:-1]] is just line[1:-1], right?

Reply:
Correct. made the change